Server/tests/articles.test.js

```diff
@@ -55,4 +57,57 @@ describe("Articles routes", () => {
     expect(res.body.message).toBe("Method Not Allowed");
     done();
   });
+
+  it("should update an article", async (done) => {
+    const res = await request.patch(`/api/articles/${id}`).send({
+      title: "Post Updated",
+      content: "Lorem ipsum Lorem ipsum Lorem ipsum",
+      imageUrl: "https://picsum.photos/200/300",
+    });
+
+    expect(res.status).toBe(201);
+    expect(res.body.message).toBe("Article updated successfully");
+    done();
+  });
+
+  it("should fail to update an article with invalid id", async (done) => {
+    const res = await request
+      .patch(`/api/articles/f322a26e079717269f2710a`)
+      .send({
+        title: "Post Updated",
+        content: "Lorem ipsum Lorem ipsum Lorem ipsum",
+        imageUrl: "https://picsum.photos/200/300",
+      });
+
+    expect(res.status).toBe(500);
```

Comment:
@elvisiraguha 500 means ` Internal Server Error` passing invalid id is not an internal server error. It means the server could not find that ID. Update elsewhere it's applicable.